wifi-connect-exclave/Dockerfile.template

```diff
@@ -1,41 +1,69 @@
 # Fin image contains → 1 container (Exclave Testing) →  Exclave (requires rust), node, NPM, codecept, balenaSDK, testbotSDK, balenaCLI, credentials for the CLI and NPM token in secrets
 
+FROM debian:buster
```

Comment:
If we are going debian:buster, would [fincm3-debian-node](https://hub.docker.com/r/balenalib/fincm3-debian-node) be a better choice instead. Or go with debian:buster-slim maybe?

Reply:
Either of those would probably work and might only require slight adjustments to the dependencies we need to install.

Reply:
Cool, our image gives us cool things like multi-device build and install_packages so that's as well. But, debian slim will be a good alternative considering we will end up shaving about 300 ish MB.

Reply:
Yeah, I'm just using this for now cause it's faster to test on my workstation with a multiarch base image. We can always change it later.